app/assets/stylesheets/osu.scss

```diff
@@ -523,6 +523,11 @@ ol.dd-list.dd-dragel {
   color: #808080;
 }
 
+.comment-line {
+  margin-top: -10px;
```

Comment:
Instead of using a negative margin, try using a 0 margin on the `.comment-time` class

Reply:
<img width="572" alt="Screenshot 2022-11-07 at 11 18 29 AM" src="https://user-images.githubusercontent.com/111306182/200395823-011618d1-078a-43bd-93c2-edf981a2b01e.png">

If I didn't add in the -10px, this is what the 0 margin-top does to the comment.

Reply:
```
.comment-time {
  font-size: 10px;
  font-style: italic;
  color: #808080;
  ~> margin-bottom: 0; <~
}
```

Reply:
Ah I see, adding the 0px to the timedate for it to push up the comment, I will push in that new update now.